package.json

```diff
@@ -15,7 +15,6 @@
     "preact-router": "^3.1.0"
   },
   "devDependencies": {
-    "@types/enzyme": "^3.10.3",
```

Comment:
This dep is the root of all pain and suffering when it comes to TS & Preact. It breaks libs and test suites because of an evil `@types/react` dependency. 

Just had to remove it to fix the suite, otherwise had silly React vs Preact type issues. It also breaks another change I have in mind for later.

Reply:
Or maybe it wasn't the issue. Well, it'll eventually cause issues.